src/main/scala/entities/course/Course.scala

```diff
@@ -0,0 +1,15 @@
+package entities.course
+
+import entities.School
+import entities.module.ModuleFehqLevel
+import entities.people.Person
+
+import scala.collection.mutable.ListBuffer
+
+class Course(
+	val courseId: Int,
+	var courseName: String,
+	var school: School,
+	var coursePeople: ListBuffer[(CourseRole, Person)],
```

Comment:
I'm not sure about slick but in frameworks like hibernate you do not need to store both people attending a course in course and the courses people are attending in person. This causes an error in fact.

Reply:
Leaving as it is as this may not be a problem if it is we'll deal with it when it comes up